components/menus/destinations.ts

```diff
@@ -420,13 +418,18 @@ export function getDestination(
         (subLocation) => subLocation.Properties.ParentLocation === LOCATION,
     )
 
-    if (query.difficulty === "pro1") {
-        type Cast = keyof typeof controller.missionsInLocations.pro1
+    // I know it's redundant to check game version here, but it's just for typescript.
+    if (query.difficulty === "pro1" && gameVersion === "h1") {
```

Comment:
For safety lets do an assertion and remove the gameVersion check

Reply:
No arguments from me.